vault_resources.go

```diff
@@ -36,8 +36,12 @@ type VaultResources struct {
 func (r *VaultResources) Set(value string) error {
 	rn := defaultVaultResource()
 
-	// step: split on the ':'
-	items := strings.Split(os.ExpandEnv(value), ":")
+	// step: split on the separator, default ':'
+	sep := os.Getenv("VAULT_SIDEKICK_SEPARATOR")
```

Comment:
we could probably reuse the [getEnv](https://github.com/UKHomeOffice/vault-sidekick/blob/master/utils.go#L144) here ... 
```Go
items := strings.Split(os.ExpandEnv(value), getEnv("VAULT_SIDEKICK_SEPARATOR", ":"))
```
or if preferred 
```Go
sep := getEnv("VAULT_SIDEKICK_SEPARATOR", ":")
``